### PR TITLE
boards: arm: holyiot_yj16019: use LFRC RC oscillator

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
The 32.768 kHz crystal shown on the Holyiot YJ-16019 schematics is not mounted. Enable the LRRC RC oscillator instead.

Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

### CODE REVIEW

**Comment by jrsharp** [comment]
This resolved the issues I encountered while testing the samples on this board. Thanks for the fix, @henrikbrixandersen !

**Review by galak — APPROVED** [review]

File: boards/arm/holyiot_yj16019/doc/index.rst [hunk 1]
```diff
@@ -34,8 +34,8 @@ contains the processor's information and the datasheet.
 Hardware
 ********
 
-The nRF52832 of the Holyiot YJ-16019 is clocked by an external crystal
-with a frequency of 32.768 kHz.
+The nRF52832 of the Holyiot YJ-16019 is clocked by an external crystal with a frequency of 32 MHz
+(Y1). The 32.768 kHz crystal (Y2) shown on the board schematics is not mounted.
```
Comment (anangl) [inline boards/arm/holyiot_yj16019/doc/index.rst:38]:
```suggestion
(X1). The 32.768 kHz crystal (Y2) shown on the board schematics is not mounted.
```

**Review by anangl — APPROVED** [review]

File: boards/arm/holyiot_yj16019/doc/index.rst (hunk 1 above)
Comment (henrikbrixandersen) [reply boards/arm/holyiot_yj16019/doc/index.rst:38]: But the part is designated Y1 on the schematics?

Comment (anangl) [reply boards/arm/holyiot_yj16019/doc/index.rst:38]: I looked at the schematic downloaded from [here](http://www.holyiot.com/ebxw_list.asp?pages=4&xwlb_id=) and it said X1:
yj-16019
so I thought that this Y1 was a typo.

**Pull request merged**